keps/sig-cli/3515-kubectl-explain-openapiv3/README.md

```diff
@@ -384,13 +338,15 @@ specified by the user's path.
 
 1. User types `kubectl explain pods`
 2. kubectl resolves 'pods' to GVR core v1 pods using cluster discovery information
-3. kubectl fetches `/openapi/v3/<group>/<version>`
-4. kubectl parses the result as kube-openapi spec3
-5. kubectl locates the schema of the return type for the Path `/apis/<group>/<version>/<resource>` in kube-openapi
-6. If a field path was used, kubectl traverses the definition's fields to the subschema
+3. kubectl attempts to fetches `/openapi/v3` which indexes where to find specs for each GV
+4. If failure and fallback to v2 is allowed, falls back to Step #3 of the "Current High-level Approach".
+5. Otherwise, kubectl fetches OpenAPIV3 path for GV: `/openapi/v3/<group>/<version>`
+6. kubectl parses the result as map[string]any
+7. kubectl locates the schema of the return type for the Path `/apis/<group>/<version>/<resource>`
+8. If a field path was used, kubectl traverses the definition's fields to the subschema
 specified by the user's path.
-8. kubectl renders the type using its built-in template for human-readable plaintext
-9. If `--recursive` was used, repeat step 8 for the transitive closure of object-typed fields of the top-level object. Concat the results together.
+9. kubectl renders the type using its built-in template for human-readable plaintext
+10. If `--recursive` was used, repeat step 8 for the transitive closure of object-typed fields of the top-level object. Concat the results together.
```

Comment:
nit: not step 8 anymore?

Reply:
Not sure what you mean here. I added a new step #3 so renumbered everything

Reply:
I mean:
```suggestion
10. If `--recursive` was used, repeat step 9 for the transitive closure of object-typed fields of the top-level object. Concat the results together.
```
unless the reference was wrong before

Reply:
whoops. nice catch